contracts/src/v0.8/dev/vrf/VRFCoordinatorV2Plus.sol

```diff
@@ -673,9 +673,15 @@ contract VRFCoordinatorV2Plus is VRF, SubscriptionAPI {
     deleteSubscription(subId);
     IVRFCoordinatorV2PlusMigration(newCoordinator).onMigration{value: ethBalance}(encodedData);
     require(LINK.transfer(address(newCoordinator), balance), "insufficient funds");
+
+    // despite the fact that we follow best practices this is still probably safest
+    // to prevent any re-entrancy possibilities.
+    s_config.reentrancyLock = true;
```

Comment:
should we also add nonReentrant() modifier to this function? Doing so will prevent other external call (e.g. callWithExactGas) from re-entering migrate()

Reply:
Yes, that makes sense.